src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.System/Main.cs

```diff
@@ -185,6 +193,23 @@ private List<Result> Commands()
                     },
                 },
             });
+
+            // UEFI command/result. It is only available on system booted in UEFI mode.
+            if (isBootedInUefiMode)
+            {
+                results.Add(new Result
+                {
+                    Title = Resources.ResourceManager.GetString(nameof(Resources.Microsoft_plugin_sys_uefi), culture),
+                    SubTitle = Resources.ResourceManager.GetString(nameof(Resources.Microsoft_plugin_sys_uefi_description), culture),
+                    FontFamily = "Segoe MDL2 Assets",
+                    Glyph = "\xE945",           // E945 => Symbol LightningBolt
```

Comment:
Why this one? Maybe "PowerButton" or something?

Reply:
It's a place holder. I like to use the official one from WinRE.